js/post_cards.js

```diff
@@ -0,0 +1,139 @@
+    This class will make the next ten start from the start of the next column .*/
+  const span3= (i+1) % 10 === 0 ? " span3":"";
+
+   $(content).append(
+      "<div class='post-prev-img " +  positions[i] + span3 + "'>" +
+        "<img class='p-cards' src='" +
+        cards[i + currentIndex].img +
+        "'  alt='' />" +
+        "</div>"
+    );
+  }
+  currentIndex += maxResult;
+
+  // Check if the 30 cards rendered and it will hide the button.
+  if (currentIndex >= cards.length) {
+    loadMoreButton.innerHTML="Go back to the top 10"
+    loadMoreButton.removeEventListener("click",loadMore,false)
+    loadMoreButton.addEventListener("click",goBack)
+
+  }
+  
+  //Get each ten cards after the first load and clicking add event listener to them.
+  var img = document.querySelectorAll(".p-cards");
+  for (let i = 0; i < img.length; i++) {
+    img[i].addEventListener("click", showImg);
+  }
+}
+
+//This function to go back to the first ten cards
+function goBack(){
+  let img = document.querySelectorAll(".post-prev-img");
+  //Go back to the top of the target id
+  document.getElementById('top-image').scrollIntoView();
+
+    for (let i = 10; i < img.length; i++) {
+      //remove the last 20 cards.
+       img[i].remove()
+    }
+       //let h = document.getElementById("cc").href="#"+firstImageLink; 
+      loadMoreButton.innerHTML="Load more postcards"
+      loadMoreButton.removeEventListener("click", goBack,false);
+      //Reset the currentIndex to 10
+      currentIndex=10;
+      loadMoreButton.addEventListener("click",loadMore);
+
+}
+//This function will view the image.
+function showImg() {
+  modal.style.display = "block";
+  modalImg.src = this.src;
+
+}
+
+// This function is to close the modal when you click x.
+span.onclick = function () {
+  modal.style.display = "none";
+};
+
+//THis function is to close the image model when you click on the space or on the image.
+modal.onclick = function () {
+  modal.style.display = "none";
+};
+
+//THis function is to close the image model when you click on the space or on the image.
+span2.onclick = function () {
+  modal2.style.display = "none";
+};
+
+//THis function is to close the vide model when you click on the space or on the image.
+modal2.onclick = function () {
+  modal2.style.display = "none";
+};
+
+//When you click the button, it will call the loadTimeLapse function.
+timeLapseButton.addEventListener("click", loadTimeLapse);
+
+//This variable tracks clicks and ensure to render the video once when you click the button
+ let counter = 0
+
+ //loadTimeLapse will append video tag to the modal
+function loadTimeLapse() { 
+   modal2.style.display = "block";
+
+  if(counter <1) {
+     counter =counter+1;
+ $(modal2).append(
+      "<video controls class='video-show' "+">" +
+        "<source src='https://cdn.bitdegree.org/learn/Pexels%20Videos%203373.mp4?raw=true' type='video/mp4'" +">"+
+        "</video>"
+    );
+  } else {
```

Comment:
small thing but could you please add to this conditional here so ``` } else if (video) {``` because we are currently getting an error in the console caused by this being hit when the video doesn't exist. 
<img width="504" alt="Screenshot 2021-03-01 at 09 22 11" src="https://user-images.githubusercontent.com/42038541/109485031-1496f100-7a79-11eb-8fac-1a85b34b5d36.png">

Reply:
I moved the video tag from the js file to html.